playground/frontend/playground_components/pubspec.lock

```diff
@@ -0,0 +1,873 @@
+# Generated by pub
```

Comment:
What is this file? At least, this file doesn't have Apache License comment above.